### ISSUE TITLE
Sketch file for (almost) entire project

### ISSUE THREAD

**Issue by cieslakdawid** [GitHawkApp/GitHawk#2025]
What do you think about including in the repo(or sharing elsewhere) Sketch file(s) that contains:
- current designs for all screens + most use-cases there (like - no data, full list, popup opened, ..) 
- all icons (if they are not included in octicons already) 
- logo
❓

It would be great to have some starting point whenever UI related changes are being made and discussed  🎉 

Pros:
- It’s much easier to brainstorm and work on UI changes when you have a wide perspective and see changes next to all other screens
- Easier to build an app with a clear, unified look
- Easier prototyping <3 
- An easy way to generate assets
- Easier to identify “areas of impact” of fix/new feature to implement
- Whenever there is someone new here, it’s easier to understand the scope of the project (for the same reason, making UI tests + generating screenshots would be great)
- cleaner "/Designs" folder 😉

Cons:
- it would probably make sense to setup screenshot tests and be strict about keeping them in sync with designs
- The process of development UI related features would be slower. You have to update the Sketch and then make a fix
- quite a lot of work to make this Sketch file 😓

With the current “blitz-development” and huge backlog it might be hard to maintain,  anyway I’m thinking about an easier way to make/share/discuss UI changes.

**Comment by rnystrom** [comment]
I’d looooooove to do this. I’ve only made 2 files, but would be happy to make a new repo for them

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cieslakdawid** [comment]
Great, I’ll check what’s possible, what makes sense now and will be back with my findings - probably on Monday 👍

**Comment by rhysforyou** [comment]
The designers at my current gig have had a lot of good things to say about [Abstract](https://www.goabstract.com/) when it comes to collaborating on designs. I think they offer free plans for OSS?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cieslakdawid** [comment]
The abstract seems great, but when I was checking their website there was nothing about OSS :(  Anyway, it's worth to ask them :)

**Comment by cieslakdawid** [comment]
So, so, so:
I didn't get a reply from Abstract about their support for OSS projects, but I've found an interesting app for design version control https://kactus.io (btw. it's built by one of the engineers working on Sketch). Free for OSS!

I've started moving some things to single `Sketch` project to see how it goes. So far just a few screens, but seems doable and I think that in ~7-10 days I would be able to share the project with most of the screens. Far from perfect, but being quite good "starting point" for collaboration :)
- it would be v. close, but not "pixel-perfect" with an app. It would require some sync in the future.
- rnystrom already mentioned that - separate repo would be awesome 🎉 + README file explaining how to play with designs, change and break things 🤘🏼
- designs need some "default" device. I was thinking that iPhone X in portrait mode might be the best for now, but I don't have strong opinion here. 

![s mov](https://user-images.githubusercontent.com/4325222/43744693-7a877254-99db-11e8-9a0e-2fbf036901e7.gif)

**Comment by Huddie** [comment]
How about minimizing the icons. Feels a bit cleaner/less cluttered this way IMO

![group 3](https://user-images.githubusercontent.com/10662471/43997776-533f1a22-9db3-11e8-863e-a46cff9d9260.png)

**Comment by rnystromtest** [comment]
My 2c, I think that makes it _more_ cluttered. Less white space.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cieslakdawid** [comment]
Hello! As I mentioned in a different thread I was playing with wireframes this weekend. Most of the screens should be here, but still "PR Details", "Code Details", (..) and many details are missing. I'm posting the current state of work, so maybe someone will find it useful now. Sorry it takes so long it ended up being quite a big task. 

- I didn't include the Text Style guide since I had some problems to identify a group of the same typefaces+sizes across the whole app. Also seems like in Mobile App the same font might have different compression.
- I took details like Icons from rnystorm's Sketch file, so thanks for that
- Icon also has been moved, so it's a bit easier to see how it looks 
- Most of the cells, headers, bars are made as symbols so it's easier to reuse and customize them
- Symbols need some cleanup
- Icon also has been moved to Sketch

<img width="921" alt="screen shot 2018-08-27 at 15 59 09" src="https://user-images.githubusercontent.com/4325222/44664187-ccfbfe00-aa12-11e8-96e0-1db1306e8ffb.png">


[GitHawk.zip](https://github.com/GitHawkApp/GitHawk/files/2324148/GitHawk.zip)
